src/test/fixtures/profiles/make-profile.js

```diff
@@ -144,18 +144,22 @@ export function getEmptyThread(overrides: ?Object): Thread {
  *
  * ```
  * const { profile } = getProfileFromTextSamples(`
- *   A       A        A     A
- *   B.js    B.js     F     F
- *   C.js    C.js     G     G
- *   D       D              H
- *   E       E
+ *   A          A         A             A
+ *   B.js       B.js      F [Graphics]  F
+ *   C.js       C.js      G             G
+ *   D          D                       H [DOM]
+ *   E [Other]  E [Other]
```

Comment:
I was going back and forth on this myself. One reason I did it was that we have a category which has spaces in the category name itself: "GC / CC". The other reason I wanted to do it is because it allows more extensions to this format without having to squish them all together; for example, we might want to add ways to have a func with multiple frames (where each of the frame has a different frame address), or we might want to explicitly specify the JS implementation on a frame.

Reply:
I see, makes sense, thanks !